release/mixer.sh

```diff
@@ -53,18 +53,34 @@ build_bundles() {
 
     log_line "Building Bundles:"
     log_line
-    sudo_mixer_cmd build bundles
+    local mixer_opts_bundles=""
+    if function_exists sign_update; then
+        mixer_opts_bundles+=" --no-signing"
+    fi
+    # shellcheck disable=SC2086
+    sudo_mixer_cmd build bundles ${mixer_opts_bundles}
     log_line
 }
 
 build_update() {
     local mix_ver="$1"
 
     section "Build 'Update' Content"
+    local mixer_opts_update=""
     if "${MIN_VERSION}"; then
-        sudo_mixer_cmd build update --skip-format-check --min-version="${mix_ver}"
-    else
-        sudo_mixer_cmd build update --skip-format-check
+        mixer_opts_update+=" --min-version=${mix_ver}"
+    fi
+    if function_exists sign_update; then
+        mixer_opts_update+=" --no-signing"
+    fi
+    # shellcheck disable=SC2086
+    sudo_mixer_cmd build update --skip-format-check ${mixer_opts_update}
+    if function_exists sign_update; then
```

Comment:
Add blank line before the if since the result of previous operation is not being evaluated here. Makes code easier to read.

Reply:
Hmm, I didn't originally include one because I ran into issues with update signing.  Because `mixer build update` requires `sudo`, this part here also requires `sudo`.  `sign_update` is a little ugly in this sense since it would also need to use `sudo` just as mixer does internally.  I was on the fence whether it should be called `sudo_sign_update`, like how we have sudo_mixer_cmd.

Reply:
Added blank line